tests/unit/amo/components/TestSearchFilters.js

```diff
@@ -108,6 +108,18 @@ describe(__filename, () => {
     });
   });
 
+  it('checks value when there is no operatingSystem filter', () => {
```

Comment:
Maybe we should dispatch an redux action to set `userAgentInfo` and use that. This is not really clear to me what this test case does. It is useful to know that the default value will be an empty string when there is no filter and, I guess, no `userAgentInfo` but we need another test case with `userAgentInfo` set to something I believe.